packages/template-retail-react-app/app/components/header/index.jsx

```diff
@@ -289,6 +307,7 @@ const Header = ({
                         {...styles.icons}
                         onClick={onMyCartClick}
                     />
+                    {!isDesktop && <SearchBar />}
```

Comment:
Optional: consider adding a comment of why we have search bar in different places for mobile vs desktop as it's not immediately clear